js/foundation.interchange.js

```diff
@@ -146,29 +163,39 @@ class Interchange extends Plugin {
   replace(path) {
     if (this.currentPath === path) return;
 
-    var _this = this,
-        trigger = 'replaced.zf.interchange';
+    var trigger = 'replaced.zf.interchange';
+
+    var type = this.options.type;
+    if (type === 'auto') {
+      if (this.$element[0].nodeName === 'IMG')
+        type = 'src';
+      else if (path.match(/\.(gif|jpg|jpeg|png|svg|tiff)([?#].*)?/i))
```

Comment:
`jpe?g` could be also used. But what about webp and other formats?

Reply:
I kept the same regex as before. But you're right, any supported `<img>` format should be added here.